general-knowledge/ethereum-blockchain/what-is-contract-abi.md

```diff
@@ -0,0 +1,17 @@
+---
+title: What is contract ABI?
+tags:
+  - Contracts
+date_published: '2021-10-29'
+date_modified: '2021-10-29'
+---
+
+An Application Binary Interface (ABI) allows the MyCrypto interface to get very basic information about the functions and function types that are available in a smart contract. It is able to show functions in a way that is minimal and comprehensible. Without ABI we wouldn't be able to know if you want to `withdrawDeFiToken(amountOfDeFiToken)` or `setOwner (address)` or both, or neither.
```

Comment:
add comma to "Without ABI, we"
add 2 commas: "`withdrawDeFiToken(amountOfDeFiToken)`, or `setOwner (address)`, or both, or neither."